src/modules/FileLocksmith/FileLocksmithUI/MainWindow.xaml.cs

```diff
@@ -14,13 +17,24 @@ public MainWindow(bool isElevated)
         {
             InitializeComponent();
             mainPage.ViewModel.IsElevated = isElevated;
-            SetTitleBar();
+            ExtendsContentIntoTitleBar = true;
```

Comment:
Nit: I'd keep these in `SetTitleBar()` (maybe rename to `SetTitleBarAndIcon()` so we don't clutter the `InitializeComponent` too much

Reply:
I don't understand. First, it would call a method with two actions, second, it's not uncommon to have a bunch of code in MainWindow() ?

Reply:
No, but the 4 lines after L19 are related to it. Better to add that to a specific method that you call in the constructor. It's easier to see what's happening then - and if you quickly want to comment it out (for debugging purposes) that's easier as well.